src/base/utils/string.h

```diff
@@ -49,6 +47,14 @@ namespace Utils
         bool naturalCompareCaseInsensitive(const QString &left, const QString &right);
 
         QString wildcardToRegex(const QString &pattern);
+
+        template <typename T>
+        T unquote(const T &str)
+        {
+            if (str.startsWith('\"') && str.endsWith('\"'))
```

Comment:
> Now it seems `Utils::String::unquote` should move to `Utils::Net::unquote`... what do you say? 

Sorry, I can't follow you. Why it is so?

> …double quote is not allowed as value in most cases (so better strip it).

Dropping invalid values differs from unquoting, doesn't it? Why don't we keep `unquote()` clean and drop invalid values afterwards? Also apparently it is needed to specify quote charachters as function parameter, because cookies can not be single quoted?